benches/oneshot/ipa.rs

```diff
@@ -34,9 +35,16 @@ async fn main() -> Result<(), Error> {
     let start = Instant::now();
     let result = world
         .semi_honest(records, |ctx, input_rows| async move {
-            ipa::<Fp32BitPrime>(ctx, &input_rows, 20, PER_USER_CAP, MAX_BREAKDOWN_KEY)
-                .await
-                .unwrap()
+            ipa::<Fp32BitPrime>(
+                ctx,
+                &input_rows,
+                20,
```

Comment:
How does this map on to #409?

Reply:
#409  is still calling generate_permutation and not generate_permutation_opt inside generate_permutation_and_reveal_shuffled. Once it is landed will stack this on top of that